src/User32.Desktop/User32.cs

```diff
@@ -1488,7 +1508,7 @@ public static partial class User32
         ///     If a message is available, the return value is true.
         ///     <para>If no messages are available, the return value is false.</para>
         /// </returns>
-        [DllImport(nameof(User32), SetLastError = true)]
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
CharSet has 2 meanings :
* It tell the marshaler that strings are UTF-16 so can be copied as-is
* It use the "W" version of the function if there is both an "A" and "W" version

So here they it must be present for the second reason. Without it we risk sending  a structure with unicode values in lParam or wParam to PostThreadMessageA that might (For known by widows messages) dutifully convert them from ANSI to unicode and pass them to PostThreadMessageW.